Makefile

```diff
@@ -108,13 +106,13 @@ manifests-dev: cmd/clusterctl/examples/aws/provider-components-base-dev.yaml ##
 
 .PHONY: cmd/clusterctl/examples/aws/provider-components-base.yaml
 cmd/clusterctl/examples/aws/provider-components-base.yaml:
-	bazel build //cmd/clusterctl/examples/aws:provider-components-base $(BAZEL_DOCKER_ARGS)
+	bazel build //cmd/clusterctl/examples/aws:provider-components-base $(BAZEL_ARGS)
 	install bazel-genfiles/cmd/clusterctl/examples/aws/provider-components-base.yaml cmd/clusterctl/examples/aws
 
 .PHONY: cmd/clusterctl/examples/aws/provider-components-base-dev.yaml
 cmd/clusterctl/examples/aws/provider-components-base-dev.yaml:
-	bazel build //cmd/clusterctl/examples/aws:provider-components-base-dev $(BAZEL_DOCKER_ARGS)
-	install bazel-genfiles/cmd/clusterctl/examples/aws/provider-components-base-dev.yaml cmd/clusterctl/examples/aws
+	bazel build //cmd/clusterctl/examples/aws:provider-components-base $(BAZEL_ARGS) --define=PULL_POLICY=Always --define=REGISTRY=gcr.io/$(shell gcloud config get-value project)
```

Comment:
In addition to exposing REGISTRY and IMAGE_PULL_POLICY as env vars from make, it might be helpful to plumb through the version override as well.

Reply:
Yep, I'm going to hold off on adding features though, I want to fix the workflow first. However, I absolutely agree with you and you'll see several more PRs for bazel over the course of the day where I will address exactly this